packages/wdio-cucumber-framework/src/index.ts

```diff
@@ -544,13 +543,14 @@ const publishCucumberReport = async (cucumberMessageDir: string): Promise<void>
         return
     }
 
-    const { headers } = await got(url, {
+    const response = await fetch(url, {
+        method: 'get',
         headers: {
             Authorization: `Bearer ${token}`,
         }
     })
 
-    const { location } = headers
+    const location = response.headers.get('location')
```

Comment:
Is there ever a chance that `location` is undefined?
```suggestion
    const location = response.headers.get('location') as string
```

Reply:
So far, I've only observed the location as a string and not any other type.